src/lib/regions.ts

```diff
@@ -0,0 +1,48 @@
+import Client from '../client'
+
+interface RegionResponse {
+  name: string
+  code: string
+  latitude: number | null
+  longitude: number | null
+  gatewayAvailable: boolean
+  requiresPaidPlan: boolean
+}
+
+interface PlatformResponse {
+  requestRegion: string | null
+  regions: RegionResponse[]
+}
+
+export interface GetRegionsOutput {
+  platform: PlatformResponse
+}
+
+const getRegionsQuery = `query {
+  platform {
+    requestRegion
+    regions {
+      name
+      code
+      latitude
+      longitude
+      gatewayAvailable
+      requiresPaidPlan
+    }
+  }
+}`
```

Comment:
Can we add a reference link to where this query is extracted from? For the other apis, we try our best to use the same query as defined by flyctl. For eg. https://github.com/superfly/flyctl/blob/master/api/resource_regions.go#L34

Reply:
Ah I see it's linked in your PR description https://github.com/superfly/flyctl/blob/master/api/resource_platform.go